### DISCUSSION TITLE
Different CRC each time I dump Famicom games

### DISCUSSION THREAD

**Discussion by Veridor** [sanni/cartreader#644] [Need help]
Every time I try to dump a Famicom game, it fails and I get a different CRC value each time. Most games dump fine with an adapter in the NES slot though. 

While one of the pins looks off on the Famicom slot, I was wondering if there were any logs I could check or post for reference.

**Comment by Veridor** [comment]
Here is the suspicious pin. 

![6A69D4DD-3AFF-4775-AD0C-297BE3DC7363](https://user-images.githubusercontent.com/1075624/204075365-ec61fa8d-2de5-499f-bca2-d3931cad735f.jpeg)

**Comment by nsx0r** [comment]
which game is it?
can you dump other FC games successfully?

**Reply by Veridor** [reply]
The main one I have issue with is Kid Dracula (Akumajō Special: Boku Dracula-kun), but so far the only way I can dump FC games is with an adapter in the NES slot.

**Reply by nsx0r** [reply]
do you get a good dump of Kid Dracula when using a NES adapter or still random CRC?

**Reply by Veridor** [reply]
That one still fails even with the adapter.

**Reply by Veridor** [reply]
![image](https://user-images.githubusercontent.com/1075624/204726768-7210dc4b-c6b2-43be-9fd3-020a2e0c4d85.jpg)

**Reply by Veridor** [reply]
At least the CRC is consistent when using the adapter

**Reply by nsx0r** [reply]
maybe the dump script is wrong, can you try with this https://github.com/sanni/cartreader/raw/4696c8d63d8b6f70ca9805978d1b969276e86351/Cart_Reader/NES.ino ?

**Reply by Veridor** [reply]
> maybe the dump script is wrong, can you try with this https://github.com/sanni/cartreader/raw/4696c8d63d8b6f70ca9805978d1b969276e86351/Cart_Reader/NES.ino ?

Do I just drag and drop that file o my SD Card, or flash using Arduino IDE? If the latter, will ha overwrite any existing code on the Arduino Mega?

**Reply by Veridor** [reply]
Nevermind, I see in the Arduino IDE Portable folder structure where I'd have to replace the nes.ino in the sketchbooks and re-flash. I'm doing a diff/compare in Notepad++ and see quite a few differences, so I wonder what changed...
